src/components/tooltip/tooltip-content.scss

```diff
@@ -9,9 +11,9 @@
 
     // To ensure that the tooltip is font-agnostic,
     // but still displays the same font as the rest of the components.
-    // This is needed, since the tooltip opens in the Portal,
-    // which inherits its font from the root element (Kompendium).
-    font-family: var(--kompendium-example-font-family, inherit);
+    // This is needed, since the tooltip opens in a portal,
+    // which inherits its font from the root element.
+    @include mixins.font-family(sans-serif);
```

Comment:
this won't work, because the component is not font-agnostic anymore now; as you are telling it which font family it should use

Reply:
can't we dynamically set `font-family: var(--kompendium-example-font-family)` on the portal itself somehow?

Reply:
another alternative is also just not to care that portal inherits its font family from root. Litterally, NOBODY will notice this.

Reply:
> this won't work, because the component is not font-agnostic anymore now; as you are telling it which font family it should use

🙈

Reply:
I removed this fixup, but added another one instead, which ought to solve the problem properly: [`968fe8b` (#2673)](https://github.com/Lundalogik/lime-elements/pull/2673/commits/968fe8b15b84b6c3cdda6e9ce0c24f81b9e26020) ([perma-link](https://github.com/Lundalogik/lime-elements/commit/968fe8b15b84b6c3cdda6e9ce0c24f81b9e26020))